docs/getstarted.md

```diff
@@ -411,3 +411,68 @@ fig.update_layout(
     title='Loss vs epochs'
 )
 ```
+
+## Run a pre-trained model on new data
+
+If you want to run a pre-trained model on new PDB files, the first step is to process and save them into HDF5 files. Let's suppose that the model has been trained with `ProteinProteinInterfaceResidueQuery` queries mapped to graphs:
+
+```python
+from deeprank2.query import QueryCollection, ProteinProteinInterfaceResidueQuery
+
+queries = QueryCollection()
+
+# Append data points
+queries.add(ProteinProteinInterfaceResidueQuery(
+    pdb_path = "<new_pdb_file1.pdb>",
+    chain_id1 = "A",
+    chain_id2 = "B"
+))
+queries.add(ProteinProteinInterfaceResidueQuery(
```

Comment:
As this part of the docs is new in this PR, the class names didn't get updated when you merged with dev.

```suggestion
If you want to run a pre-trained model on new PDB files, the first step is to process and save them into HDF5 files. Let's suppose that the model has been trained with `ProteinProteinInterfaceQuery` queries mapped to graphs:

```python
from deeprank2.query import QueryCollection, ProteinProteinInterfaceQuery

queries = QueryCollection()

# Append data points
queries.add(ProteinProteinInterfaceQuery(
    pdb_path = "<new_pdb_file1.pdb>",
    chain_id1 = "A",
    chain_id2 = "B"
))
queries.add(ProteinProteinInterfaceQuery(
```